presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -947,7 +947,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                     tableProperties.put(CSV_SEPARATOR_KEY, separator.toString());
                 });
 
-        // prevent HMS from gathering these stats at table creation, since it can be expensive
+        // set bogus table stats to prevent Hive 2.x from gathering these stats at table creation
```

Comment:
```suggestion
        // Set bogus table stats to prevent Hive 2.x from gathering these stats at table creation.
        // These stats are not useful in itself and can take very long time to collect when creating an
        // external table over large data set.
```